src/test/resources/metafiles/valid/from-spec-no-scannable-items.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I'd remove `from-spec-` prefix from these file names, as it doesn't really say anything (and is a bit confusing to me).